### PR TITLE
used Doc2VecVocab for scan_vocab method

### PR DESCRIPTION
**Pull request by Naba7**
I have made some documentation changes along with code.
I have imported Doc2VecVocab and instantiated a variable 'pres' to call 'scan_vocab' method

### CODE REVIEW

File: docs/notebooks/doc2vec-wikipedia.ipynb [hunk 1]
```diff
@@ -158,8 +158,9 @@
    },
    "outputs": [],
    "source": [
+    "pres = Doc2VecVocab()\n",
     "pre = Doc2Vec(min_count=0)\n",
-    "pre.scan_vocab(documents)"
+    "pres.scan_vocab(documents)"
```
Comment (menshikh-iv) [inline docs/notebooks/doc2vec-wikipedia.ipynb]: why? this isn't used.

Comment (Naba7) [reply docs/notebooks/doc2vec-wikipedia.ipynb]: scan_vocab function in Doc2VecVocab class as in http://localhost:8889/edit/gensim/models/doc2vec.py
That is the reason I have created another object to call that class and dot operator to call the function scan_vocab. All other functions such as build_vocab,init_sims,etc functions are in Doc2Vec class.

Comment (menshikh-iv) [reply docs/notebooks/doc2vec-wikipedia.ipynb]: let's try again: `Doc2VecVocab` class not used and not related to actual instance of Doc2Vec -> shouldn't be here.

Comment (Naba7) [reply docs/notebooks/doc2vec-wikipedia.ipynb]: okay!

**Comment by mpenkov** [comment]
Closing as abandoned

**Pull request closed without merging**